### ISSUE TITLE
OOM when running Object_detection_image.py ?

### ISSUE THREAD

**Issue by kd97** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#254]
It's ok when running train.py,but when I run Object_detection_image.py,it goes wrong(OOM)?
It's so strange.Can anyone help me?
 Allocator (GPU_0_bfc) ran out of memory trying to allocate 2.05GiB. The caller indicates that this is not a failure, but may mean that there could be performance gains if more memory were available.

**Comment by nkvuradi** [comment]
I have seen this too and I think this is just a warning that can be ignored. I did not see any functional impact  on the model because of this.

**Comment by kd97** [comment]
@nkvuradi but I can't get the result. It does not draw any bounding boxes and detect cards.It seems to break halfway?

**Comment by kd97** [comment]
@nkvuradi you're right!After running train.py for a longer time,I see the result.Thanks a lot!

**Comment by nkvuradi** [comment]
Happy for you!
